lib/libutils/isoc/bget.h

```diff
@@ -33,7 +34,7 @@ void   *bget	    _((bufsize size, struct bpoolset *poolset));
 void   *bgetz	    _((bufsize size, struct bpoolset *poolset));
 void   *bgetr	    _((void *buffer, bufsize newsize,
 		       struct bpoolset *poolset));
-void	brel	    _((void *buf, struct bpoolset *poolset));
+void	brel	    _((void *buf, struct bpoolset *poolset, int wipe));
```

Comment:
`int` or `bool`?

Reply:
`bool` doesn't work with the pre-ANSI C prototype if I recall correctly (there is a compiler warning or error).

Reply:
ok